examples/custom-image-dataset/examples/custom-image-dataset.rs

```diff
@@ -0,0 +1,51 @@
+#[cfg(feature = "tch-gpu")]
+mod tch_gpu {
+    use burn::backend::libtorch::{LibTorch, LibTorchDevice};
+    use burn::backend::Autodiff;
+    use burn::optim::momentum::MomentumConfig;
+    use burn::optim::SgdConfig;
+    use custom_image_dataset::training::{train, TrainingConfig};
+
+    pub fn run() {
+        #[cfg(not(target_os = "macos"))]
+        let device = LibTorchDevice::Cuda(0);
+        #[cfg(target_os = "macos")]
+        let device = LibTorchDevice::Mps;
+
+        train::<Autodiff<LibTorch>>(
+            TrainingConfig::new(SgdConfig::new().with_momentum(Some(MomentumConfig {
+                momentum: 0.9,
+                dampening: 0.,
+                nesterov: false,
+            }))),
```

Comment:
The question is: is Adam better in this case 😅

Reply:
I actually haven't fiddled with the training recipe too much for this example 😄 a bunch of things could probably be changed to further optimize the results, I just wanted to keep it simple.

Lmk if you want any changes to be applied to the training example.